### PR TITLE
aup policy

### PR DESCRIPTION
**Pull request by SinenWew**
# Description 

I want to add the AUP policy to the school website

## What

Adding AUP Policy to the school website

## Why

School website needs to be updated

## How

via SublimeText

## Verification Steps

Go into school website --> about --> school policies, and ensure that AUP Policy is present

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This change has been verified by another team member

### CODE REVIEW

**Comment by SinenWew** [comment]
@laurafitzgerald @evanhearne 
Review this please? Thanks! <3

**Comment by evanhearne** [comment]
@SinenWew reviewing this now.

**Comment by laurafitzgerald** [comment]
> @SinenWew please look at the Prohibition, Appendix II and permission forms sections of the page when you have the time and please remove any other sections that have forms to fill out or anything that has to be filled out. We will look at this tomorrow.

@evanhearne can you check with @edkennedy50 about the forms. I have some memory that they are to stay in.

**Comment by evanhearne** [comment]
@laurafitzgerald I will check with him about this and get back to you. As far as the position of the policy is concerned, me and @SinenWew know about this issue and we will be investigating why this is and addressing it tomorrow.

**Comment by evanhearne** [comment]
@SinenWew reviewing changes now, will leave review in ten minutes.

**Comment by evanhearne** [comment]
@edkennedy50 says it's fine to leave it as it is.

File: pages/about/policies/AUP.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+<li>
+<p>Filtering systems (HEANET) and or equivalent systems will be used where appropriate in order to minimise the risk of exposure to inappropriate material.</p>
+</li>
+<li>
+<p>The school will regularly monitor students&rsquo; internet usage.</p>
+</li>
+<li>
+<p>Each student will be issued with a username and password for their own network account. Students must use their own account at all times.</p>
+</li>
+<li>
```
Comment (laurafitzgerald) [inline pages/about/policies/AUP.html]: @evanhearne @SinenWew some of the additional html tags here are causing the format of the text to have a lot of spacing. We saw this with previous ones as well. To make the layout look a bit better could you make changes to improve this?

Comment (evanhearne) [reply pages/about/policies/AUP.html]: @laurafitzgerald that isn't a problem. Me and @SinenWew will get to work on that on Wednesday hopefully.

**Comment by evanhearne** [comment]
> eyes reviewing this now. @evanhearne you also need to rebase this branch as there are conflicts in the file. This should be done via the command line as we did before.

@laurafitzgerald I will do this now.

**Comment by laurafitzgerald** [comment]
@evanhearne if you need some help with the rebase (I think it's a while since the team had to do one) just give me a shout.

**Comment by evanhearne** [comment]
@laurafitzgerald changes have been fixed, I had some problems rebasing but got through. When you have the time please review :)

File: pages/about/policies.html [hunk 2]
```diff
@@ -1,444 +1,634 @@
...
+                <div class="eight wide column">
+                    <div class="ui link list">
+                    <h4 class="ui inverted header">Mount Sion C.B.S Waterford</h4>
+                    <p>An all boys school located in the heart of Waterford City</p>
+                    <p>It is the founding school set up by Edmund Rice.</p>
+                    </div>
+                </div>
             </div>
-          </div>
-          <div class="column">
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: The antibullyinp policy preview was removed here. I'm not sure that was intended.

**Comment by evanhearne** [comment]
I'm closing this branch as I made it into a mess and I am going to start a new one :)

**Pull request closed without merging**